modules/nw-ingress-reencrypt-route-custom-cert.adoc

```diff
@@ -17,6 +17,44 @@ The `route.openshift.io/destination-ca-certificate-secret` annotation can be use
 
 .Procedure
 
+. Create a secret for the destination-ca-certificate
++
+[source terminal]
+----
+$ oc create secret generic dest-ca-cert --from-file=tls.crt=<file_path>
+----
++
+For example:
++
+[source terminal]
+----
+$ oc -n test-ns create secret generic dest-ca-cert --from-file=tls.crt=tls.crt
+secret/dest-ca-cert created
+----
+
+. Get the destination-ca-certificate
```

Comment:
```suggestion
. Get the destination CA certificate
```

Is this step a verification step? What information needs to be extracted for the next step?